dashboard/cron.py

```diff
@@ -48,17 +48,14 @@ def util_function(UDW_course_id, sql_string, mysql_table, table_identifier=None)
 
     # drop duplicates
     df.drop_duplicates(keep='first', inplace=True)
-    
-    logger.debug("Duplicates dropped")
-    logger.debug(df)
 
     logger.debug(" table: " + mysql_table + " insert size: " + str(df.shape[0]))
 
     # write to MySQL
     try:
         df.to_sql(con=engine, name=mysql_table, if_exists='append', index=False)
     except Exception as e:
-        logger.debug(e)
+        logger.warn(e)
```

Comment:
How about log the mysql_table name in the log file?

Otherwise, it is OK to merge

Reply:
```suggestion
        logger.exception(f"Error running to_sql on table {mysql_table}")
```